kedro/io/cached_dataset.py

```diff
@@ -27,6 +27,12 @@ class CachedDataSet(AbstractDataSet):
     class as shown above.
     """
 
+    # this dataset cannot be used with ``ParallelRunner``,
+    # therefore it has the attribute ``_SINGLE_PROCESS = True``
+    # for parallelism within a Spark pipeline please consider
+    # ``ThreadRunner`` instead
```

Comment:
We can remove these two sentences, because for the `CachedDataSet` this is not related to Spark in any way.

Reply:
Sure! Thanks for the suggestion

Reply:
@MerelTheisenQB may we could keep the suggestion to use ``ThreadRunner``?

````
# for parallelism please consider ``ThreadRunner`` instead
````